### ISSUE TITLE
Dropdown is not rendered properly in high contrast mode

### ISSUE THREAD

**Issue by noisen** [OfficeDev/office-ui-fabric-react#11807]
### Environment Information

- **Package version(s)**: 7.85.0
- **Browser and OS versions**: Edge 79.0.309.71 (64-bit) on Windows 10 Ent Version	10.0.17134

### Describe the issue:

When the high contrast mode (any HC scheme) is enabled and the Dropdown is focused with the keyboard, the component renders as a plain highlight rectangle without the text and the chevron. 
![image](https://user-images.githubusercontent.com/1035615/73187731-c434bd80-4119-11ea-9107-953fc24c624c.png)
I believe, it's the ::after pseudo-element with the content="" is covering the whole thing with the solid background.

### Please provide a reproduction of the issue in a codepen:

https://fabricweb.z5.web.core.windows.net/oufr/7.85.0/#/examples/dropdown

#### Actual behavior:

The text and the chevron are not visible

#### Expected behavior:

The text and the chevron are visible and contrast to the background

**Comment by khmakoto** [comment]
Hi @noisen, I can confirm this issue on our end, thanks for filing it! I'll have a fix as soon as I can.
